### PR TITLE
Nit: Apply suggestion for Shoot status doc

### PR DESCRIPTION
**Pull request by ialidzhikov**
**How to categorize this PR?**

/area documentation
/priority normal

**What this PR does / why we need it**:
Ref https://github.com/gardener/gardener/pull/2765#pullrequestreview-472247932.

**Release note**:

```improvement operator
NONE
```

### CODE REVIEW

**Review by rfranzke — DISMISSED** [review]

File: docs/usage/shoot_status.md [hunk 1]
```diff
@@ -73,4 +73,4 @@ Known error codes are:
 - `ERR_INFRA_DEPENDENCIES` - indicates that the last error occurred due to dependent objects on the infrastructure level
 - `ERR_INFRA_RESOURCES_DEPLETED` - indicates that the last error occurred due to depleted resource in the infrastructure
 - `ERR_CLEANUP_CLUSTER_RESOURCES` - indicates that the last error occurred due to resources in the cluster that are stuck in deletion
-- `ERR_CONFIGURATION_PROBLEM` - indicates that the last error occurred due a configuration problem
+- `ERR_CONFIGURATION_PROBLEM` - indicates that the last error occurred due to configuration problem
```
Comment (timebertt) [inline docs/usage/shoot_status.md]: I think @timuthy's suggestion was like this:
```suggestion
- `ERR_CONFIGURATION_PROBLEM` - indicates that the last error occurred due to a configuration problem
```
Same for the other doc strings.

**Review by timebertt — COMMENTED** [review]
Sorry for merging the other PR too fast...

**Review by timebertt — APPROVED** [review]

**Review by timuthy — APPROVED** [review]
/lgtm

Thanks!

**Pull request merged**